src/scikit_build_core/builder/wheel_tag.py

```diff
@@ -154,7 +154,11 @@ def as_tags_set(self) -> frozenset[packaging.tags.Tag]:
 if __name__ == "__main__":
     from .._compat.argparse import ArgumentParser
 
-    parser = ArgumentParser(allow_abbrev=False)
+    parser = ArgumentParser(
+        prog="scikit_build_core.builder.wheel_tag",
```

Comment:
A bit inconsistent with the `prog` reference between each one

Reply:
```suggestion
        prog="python -m scikit_build_core.builder.wheel_tag",
```